test/actions.test.ts

```diff
@@ -77,84 +66,113 @@ describe("Test Custom SDK Logic", () => {
     return domainData.name;
   };
 
+  describe("E2E with Rinkeby", () => {
+    it("mints a domain", async () => {
+      // private key must point to owner of parent domain, in this case wilder
```

Comment:
nit - remove this test if it's not needed

Reply:
Some tests that perform real transactions are left to show that the functionality was tested and is working as expected, but commented out to not incur costs every time they're run. I think it should stay